pkg/tms/tmsUtils_test.go

```diff
@@ -0,0 +1,47 @@
+package tms
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_unmarshalServiceKey(t *testing.T) {
+	tests := []struct {
+		name           string
+		serviceKeyJson string
+		wantTmsUrl     string
+		errMessage     string
+	}{
+		{
+			name:           "standard cTMS service key uri works",
+			serviceKeyJson: `{"uri": "https://my.tms.endpoint.sap.com"}`,
+			wantTmsUrl:     "https://my.tms.endpoint.sap.com",
+		},
+		{
+			name:           "standard cALM service key uri has expected postfix",
+			serviceKeyJson: `{"endpoints": {"Api": "https://my.alm.endpoint.sap.com"}}`,
+			wantTmsUrl:     "https://my.alm.endpoint.sap.com" + CALM_REROUTING_ENDPOINT_TO_CTMS,
```

Comment:
Please hardcode the rerouting path here. So if the constant changes, the test will detect it.